### PR TITLE
DW 1327 ADFS authentication

### PR DESCRIPTION
**Pull request by alexandru-mitcu-valtech**

### CODE REVIEW

File: site/webapp/pom.xml [hunk 1]
```diff
@@ -63,7 +63,7 @@
                 </includes>
             </resource>
             <resource>
-                <filtering>true</filtering>
+                <filtering>false</filtering>
                 <directory>src/main/resources</directory>
```
Comment (GeoffHayward) [inline site/webapp/pom.xml:67]: I take it you checked for any regressions.

Comment (alexandru-mitcu-valtech) [reply site/webapp/pom.xml:67]: I did, everything seems fine.

File: site/components/src/main/java/uk/nhs/digital/website/security/stub/HippoUserDetailsServiceStub.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package uk.nhs.digital.website.security.stub;
+
+import org.onehippo.forge.security.support.springsecurity.authentication.HippoUserDetailsService;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import uk.nhs.digital.intranet.model.IntranetUser;
+
+import java.util.Collections;
+
+public class HippoUserDetailsServiceStub implements HippoUserDetailsService {
```
Comment (GeoffHayward) [inline site/components/src/main/java/uk/nhs/digital/website/security/stub/HippoUserDetailsServiceStub.java:10]: Should this be in the test folder?

Comment (alexandru-mitcu-valtech) [reply site/components/src/main/java/uk/nhs/digital/website/security/stub/HippoUserDetailsServiceStub.java:10]: No. This is used in the local development environment, not in unit tests. It's an actual stub.

**Comment by thebib** [comment]
This branch is a historic ticket for work on the Intranet.
Intranet work is halted, this PR can be recreated if needed.

**Pull request closed without merging**